### ISSUE TITLE
Question about performance of supervised learning on target domain

### ISSUE THREAD

**Issue by jiangzhengkai** [tsunghan-wu/D2ADA#2]
Hi, I notice supervised learning on target domain only achieves 71.3% mIoU, which is much lower than proposed in mmsegmentation. Any reasons for such phenomenon?

**Comment by tsunghan-wu** [comment]
Hi @jiangzhengkai,

Thanks for your feedback on our paper. I apologize for the delay in my response due to my busy schedule. I wanted to provide some additional context and clarify a few points:

1. **Implementation details.** I just checked out the Github Repository for [MMSegmentation](https://github.com/open-mmlab/mmsegmentation/tree/master/configs/deeplabv3plus) and it works really well as you said. Our code is largely based on [THIS](https://github.com/VainF/DeepLabV3Plus-Pytorch) GitHub repository, but the performance of DeepLabV3+ in our paper is slightly lower than their reported result (73.2 vs 76.2). We believe that "freezing the batch norm layer" may be one reason for this difference. Also, as I haven't utilized MMSegmentation before, I'm not sure where the difference in their implementation is. Thanks for the info, maybe I should take a closer look...

2. **Comparison to the original papers.** In Table 7 of the original [DeepLabV3+ paper](https://arxiv.org/pdf/1802.02611.pdf), the authors used a different, deeper network backbone (X-71) to achieve 80+ mIoU. In contrast, we used the standard ResNet-101 backbone that is commonly used in prior domain adaptation work. Additionally, as shown in Table 9 of the original [DeepLabV2 paper,](https://arxiv.org/pdf/1606.00915.pdf) our reproduced result in Table 1 of our D2ADA paper (71.3 mIoU) is very close to the official Cityscapes performance (71.4 mIoU) with the same ResNet-101 backbone.

3. **Comparison to recent DA studies.** Our reported "Target Only" results are similar to the results reported in recent outstanding papers (such as [RIPU](https://openaccess.thecvf.com/content/CVPR2022/papers/Xie_Towards_Fewer_Annotations_Active_Learning_via_Region_Impurity_and_Prediction_CVPR_2022_paper.pdf) in CVPR 2022) and better than previous work (such as [LabOR](https://openaccess.thecvf.com/content/ICCV2021/papers/Shin_LabOR_Labeling_Only_if_Required_for_Domain_Adaptive_Semantic_Segmentation_ICCV_2021_paper.pdf) in ICCV 2021).

Please let us know if you have any other questions. We appreciate your feedback and will continue to work on improving our results.

Best,
Patrick
